src/wash/wash.cpp

```diff
@@ -93,13 +95,22 @@ namespace wash {
     }
 
     void start() {
+        auto& io = get_io();
+        io.set_path(simulation_name, output_file_name);
+
         for (auto& k : init_kernels) {
             k->exec();
         }
+
+        io.handle_iteration(-1);
+        
         for (uint64_t iter = 0; iter < max_iterations; iter++) {
             for (auto& k : loop_kernels) {
                 k->exec();
             }
+
+            io.handle_iteration(iter);
+            std::cout << "Finished iter " << iter << std::endl;
```

Comment:
Not sure if printing debug info here is the best idea. Maybe surround it with `#ifdef DEBUG`?

Reply:
I've found when running long-running programs especially over the batch-compute machines it's useful to have some output to just show it making progress. Not sure if it's really "debugging" in that sense.

Reply:
Okay, fair point